src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -203,12 +205,16 @@ public boolean shouldTraverse(NodeTraversal t, Node n, Node parent) {
 
   /**
    * Converts a Closure goog.require call into a TypeScript import statement.
-   *
+   * <p>
```

Comment:
By Google standards, it should be on the same line as the text, right?

```
/**
 * Foo bar baz.
 *
 * <p>Bam blergh.
 */
```

... not that I care.